umbral/keys.py

```diff
@@ -108,6 +111,43 @@ def get_pubkey(self):
         """
         return UmbralPublicKey(self.bn_key * self.params.g)
 
+    def to_cryptography_privkey(self):
+        """
+        Returns a cryptography.io EllipticCurvePrivateKey from the Umbral key.
+        """
+        backend = default_backend()
+
+        backend.openssl_assert(self.bn_key.group != backend._ffi.NULL)
+        backend.openssl_assert(self.bn_key.bignum != backend._ffi.NULL)
+
+        ec_key = backend._lib.EC_KEY_new()
+        backend.openssl_assert(ec_key != backend._ffi.NULL)
+        ec_key = backend._ffi.gc(ec_key, backend._lib.EC_KEY_free)
+
+        res = backend._lib.EC_KEY_set_group(ec_key, self.bn_key.group)
```

Comment:
I don't think I like `res` as the name of the result of every call to the backend.  Can't we have more informative and unique names?

In addition to being confusing, it's a bit dangerous: if I comment out one of the intermediate lines, the assertion still passes, right?  Don't we not want that?

Reply:
It's a `res` as much as a `result`. It either outputs `1`, `0`, or in some cases `-1`.

Reply:
Yeah, I mean I know that.  But what you do mean it's a "res as much as a result"?  As I asked above, why got have informative and unique names?  That's what I always do.

Reply:
I mean you could change `res` to `ret` or `ret_code`, but pretty much anyone who looks at this code is going to understand what `res` is.

Reply:
Sure, but those are equally dumb names.  I'm talking about actual unique names.

Reply:
`success_code`?

Reply:
I mean like this:

```
        new_ec_key = backend._lib.EC_KEY_new()
        backend.openssl_assert(new_ec_key != backend._ffi.NULL)
        ec_key = backend._ffi.gc(new_ec_key, backend._lib.EC_KEY_free)

        set_group_result = backend._lib.EC_KEY_set_group(ec_key, self.bn_key.group)
        backend.openssl_assert(set_group_result == 1)
```

Reply:
Ah.. Makes sense

Reply:
Here's a PR: https://github.com/tuxxy/pyUmbral/pull/1